### DISCUSSION TITLE
[FEATURE REQUEST] More markdown syntax and features

### DISCUSSION THREAD

**Discussion by seanmrnda** [adityatelange/hugo-PaperMod#951] [Ideas]
Markdown syntax and features similar to LoveIt theme.

[LoveIt Link](https://hugoloveit.com/theme-documentation-extended-shortcodes/#4-admonition)

![image](https://user-images.githubusercontent.com/45195351/174483287-2659616e-8cd6-492e-9cea-8b38cb8834de.png)

**Comment by adityatelange** [comment]
If loveit has all the features you need, feel free to switch to loveit 😄

**Reply by seanmrnda** [reply]
I like the simplicity of PaperMod. But adding more features is not a bad idea. ;)

**Reply by adityatelange** [reply]
Don't you think addition of new features more and more will decrease the simplicity significantly?

**Reply by seanmrnda** [reply]
Well, you are right. Maybe the option to resize the margins by using a keyword in config.yaml/config.toml that would affect site wide or a specific page margins, esp. in blog posts.

E.g. 
```
pageMarginResize: 
```
or whatever keyword.

At least having this feature would be great.

**Comment by dzintars** [comment]
You could do that with your custom [shortcodes](https://gohugo.io/templates/shortcode-templates/#custom-shortcode-examples) and custom [css](https://github.com/adityatelange/hugo-PaperMod/wiki/FAQs#bundling-custom-css-with-themes-assets).
